### DISCUSSION TITLE
Release cycle for adguard home?

### DISCUSSION THREAD

**Discussion by hexclann** [AdguardTeam/AdGuardHome#5326] [Q&A]
What's the release cycle for adguard home? v108 is in beta for a very long time and the redesign of the dashboard as stated by the developer is not seen in v108.

DoT with wildcard certificate is supported only on v108

**Comment by gene1wood** [comment]
> DoT with wildcard certificate is supported only on v108

Ya, this feature is needed to be able to use Android Private DNS
